### PR TITLE
added some more stuff to the change log. Added helpers to make change…

### PR DESCRIPTION
**Pull request by Kevin-Wei**
… log tell yo who changed stuff

It relates to the following issue #s: 
* Attempt at #319 

we need some more helpers to clean up the way the thing looks. Lots of guid hiding needed here

### CODE REVIEW

**Comment by colinxfleming** [comment]
@ajohnson052 can I tag you in to review this one?

**Comment by Kevin-Wei** [comment]
Yeah still gotta clean up tests / some other things

**Comment by ajohnson052** [comment]
Happy to look through this. @Kevin-Wei, do you want to just tag me when you're done cleaning things up?

File: app/views/pregnancies/_change_log.html.erb [hunk 1]
```diff
@@ -18,7 +18,57 @@
...
+            <td><%= revision.tracked_changes_fields %></td>
+            <td><%= revision.tracked_changes_from %></td>
+            <td><%= revision.tracked_changes_to %></td>
+            <td><%= updated_by_user(revision) %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+  <div class="col-sm-12">
```
Comment (colinxfleming) [inline app/views/pregnancies/_change_log.html.erb]: Is this block appearing twice?

File: app/helpers/change_log_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  def updated_by_user(revision)
+    if revision.tracked_changes[:updated_by_id].present?
+      user_guid_to_email(revision.tracked_changes[:updated_by_id][:to])
+    else
+      "N/A"
+    end
+  end
+
+  def user_guid_to_email(guid)
+    if User.where(id: guid).first.present?
```
Comment (colinxfleming) [inline app/helpers/change_log_helper.rb]: No `find`?

File: app/helpers/change_log_helper.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      User.where(id: guid).first.email
```
Comment (colinxfleming) [inline app/helpers/change_log_helper.rb]: Thoughts about using `name` instead of `email`?

File: app/helpers/change_log_helper.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+module ChangeLogHelper
+
+  def updated_by_user(revision)
+    if revision.tracked_changes[:updated_by_id].present?
+      user_guid_to_email(revision.tracked_changes[:updated_by_id][:to])
+    else
+      "N/A"
```
Comment (colinxfleming) [inline app/helpers/change_log_helper.rb]: Will it ever actually hit this condition, since it's defaulting to `system` in `user_guid_to_email`?

**Comment by colinxfleming** [comment]
ayo @Kevin-Wei , marked this up a bit -- let me know what yr thoughts are

**Comment by colinxfleming** [comment]
@Kevin-Wei I'm picking up this thread in #707

**Pull request closed without merging**